target-group.tf

```diff
@@ -20,3 +20,30 @@ resource "aws_lb_target_group" "this" {
     protocol            = var.protocol
   }
 }
+
+resource "aws_lb_target_group" "additional" {
+  for_each = var.additional_ports
+
+  name                 = "${local.resource_name}-${each.value}"
+  port                 = each.value
+  protocol             = var.protocol
+  target_type          = "ip"
+  vpc_id               = local.vpc_id
+  deregistration_delay = 10
+  tags                 = local.tags
+
+  health_check {
```

Comment:
Won't this create a health_check for every additional port that is configured? Don't you want just one?